tests/integration/test_labels.py

```diff
@@ -31,7 +31,7 @@ def empty_ruleset():
 
 
 def get_results_from_colin_labels_image():
-    return colin.run("colin-labels", ruleset_name="fedora")
+    return colin.run("colin-labels", ruleset_name="fedora", logging_level=10)
```

Comment:
maybe would be better to use constants inside logging module ``logging_level=logging.INFO``

Reply:
this way I don't have to import logging :P

Reply:
anyone, fixed

Reply:
little bit unclear,  but makes sense :-)